fc/parsing/actions.py

```diff
@@ -900,7 +961,21 @@ def _add_or_replace_equations(self):
             if defn is not None:
                 self.model.remove_equation(defn)
             self.model.add_equation(eq)
-        # TODO: Check units of newly added equations; apply conversions where needed?
+        # TODO: Check units of newly added equations; apply conversions where needed now or later?
+
+    def _handle_clamping(self):
+        """Clamp requested variables to their initial values."""
+        for clamp in self._clamps:
+            var = self.model.get_symbol_by_ontology_term(clamp.rdf_term)
+            defn = self.model.get_definition(var)
+            if var.initial_value is None:
+                value = defn.rhs.evalf()  # TODO: What if there are variable references in the RHS?
```

Comment:
Is this for a future test case? Will evalf() fail here at least?

Reply:
Yes, I think evalf would fail in this case. It was the question I posed on #96.